corgi/api/views.py

```diff
@@ -566,8 +566,8 @@ class ComponentViewSet(ReadOnlyModelViewSet):  # TODO: TagViewMixin disabled unt
     filterset_class = ComponentFilter
 
     def get_queryset(self) -> QuerySet[Component]:
-        # we always want a distinct result from REST API
-        return self.queryset.distinct()
+        # ensure we always return distinct set of components from REST API and enforce a common sort
+        return self.queryset.order_by("name").distinct()
```

Comment:
Justin said on [pull/632](https://github.com/RedHatProductSecurity/component-registry/pull/632#pullrequestreview-1671346151) that "sort order must be on either a field with unique=True, or all fields appearing together in a UniqueConstraint, which should be the case today. " Therefore candidates are "uuid", "purl" or the combination of ("name", "type", "arch", "version", "release"). If not we will likely run into bugs paginating results as OSIDB did.

Reply:
ah right ... let me investigate - will probably remove and keep status quo